Libraries/Components/ScrollView/ScrollViewStickyHeader.js

```diff
@@ -18,7 +18,7 @@ import type {LayoutEvent} from '../../Types/CoreEventTypes';
 
 const AnimatedView = AnimatedImplementation.createAnimatedComponent(View);
 
-type Props = {
+export type ScrolViewStickyHeaderProps = {
```

Comment:
typo: ScrolView

Reply:
I think I would prefer calling this `Props` and then using `import type {Props as ScrollViewStickyHeaderProps}` where it is being imported.

Reply:
@cpojer done